programs/keeper-client/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+set (CLICKHOUSE_KEEPER_CLIENT_SOURCES KeeperClient.cpp)
+
+set (CLICKHOUSE_KEEPER_CLIENT_LINK
+    PRIVATE
+    boost::program_options
+    dbms
+)
+
+clickhouse_program_add(keeper-client)
```

Comment:
would be nice to have it as a standalone binary so maybe think early about minimizing included libs?
This is just a suggestion so feel free to do it in any way you think it's easiest for you.

Reply:
@antonio2368 Initial idea was to include this client only to standalone `keeper` and bigger `clickhouse` binary

Reply:
that also works